### PR TITLE
Add IMU timeline documentation to Player docs

### PR DESCRIPTION
**Pull request by N-M-T**

### CODE REVIEW

File: src/core/software/pupil-player.md [hunk 1]
```diff
@@ -236,7 +236,47 @@ Results are exported in the following files:
 - `head_pose_tacker_model.csv`: A list of all markers used to generate the 3d model and the 3d locations of the marker vertices.
 - `head_pose_tacker_poses.csv`: The world camera's pose within the 3d model coordinate system for each recorded world frame. A camera pose is described as a 6-components vector. The first three components are the rotation vector in Rodrigues format and the last three components are the translation vector.
 
+#### IMU Timeline
+This plugin visualizes accelerometer and gyroscope data from Pupil Invisible recordings. It also fuses the 
+data using Madgwick's algorithm to yield drift-free orientation estimates in the pitch and roll axes.
```
Comment (papr) [inline src/core/software/pupil-player.md:241]:
```suggestion
data using [Madgwick's algorithm](https://github.com/xioTechnologies/Fusion) to yield drift-free orientation estimates in the pitch and roll axes.
```

File: src/core/software/pupil-player.md [hunk 2]
```diff
@@ -236,7 +236,47 @@ Results are exported in the following files:
...
+| `accel_z`           | Linear acceleration along the z-axis in G (9.80665 m/s<sup>2</sup>)    |
+| `pitch`             | Orientation about the x-axis (head tilt from front to back) in degrees |
+| `roll`              | Orientation about the z-axis (head tilt from side to side) in degrees  |
+
+:::tip
+<v-icon large color="info">info_outline</v-icon>
+Read more about [Pupil Invisible's coordinate systems here](/developer/invisible/#coordinate-systems).
+:::
+
+This Plugin is does not estimate orientation about the yaw axis (head rotation from left to right). This is
```
Comment (papr) [inline src/core/software/pupil-player.md:275]:
```suggestion
This Plugin does not estimate orientation about the yaw axis (head rotation from left to right). This is 
```

**Review by papr — COMMENTED** [review]
So from what I understand Sebastian Madgwick authored the original algorithm but xioTechnologies is now maintaining it. Maybe, we should give credit to both? In any case, I suggest to link to their official github repo, see below.

**Comment by willpatera** [comment]
@papr @N-M-T are we awaiting further changes or shall I review?

**Comment by N-M-T** [comment]
@papr I applied your suggestions, and updated and added a link to the Player Specific Plugin table. I don't think there is anything else to add, in which case it's ready for your review @willpatera

**Pull request merged**